backend/tests/services/invoice/invoiceServiceFactory.test.js

```
@@ -0,0 +1,79 @@
+}));
+jest.mock('../../../src/services/s3Service');
+
+describe('createInvoiceService Factory Function', () => {
+  // Save original environment
+  const originalEnv = process.env;
+  
+  beforeEach(() => {
+    jest.clearAllMocks();
+    // Reset process.env to clean state for each test
+    process.env = { ...originalEnv };
+    delete process.env.OCR_ANALYZER_TYPE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)